server/src/game/role/wizard.rs

```diff
@@ -0,0 +1,99 @@
+use serde::Serialize;
+
+use crate::game::chat::{ChatGroup, ChatMessage};
+use crate::game::phase::PhaseType;
+use crate::game::player::PlayerReference;
+use crate::game::role_list::Faction;
+use crate::game::visit::Visit;
+use crate::game::Game;
+use crate::game::team::Team;
+use super::{Priority, RoleStateImpl};
+
+pub(super) const FACTION: Faction = Faction::None;
+pub(super) const MAXIMUM_COUNT: Option<u8> = None;
+
+#[derive(Clone, Debug, Serialize, Default)]
+pub struct Wizard{
```

Comment:
Why is this in this pull request? Seems unrelated.
Make another PR

Reply:
There is so much broken with this role, I'm not going to review it right now. Just remove it and make another pull request